src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobDispatcherTask.java

```diff
@@ -36,17 +39,20 @@ public BlobDispatcherTask(
     public void run() {
         logger.info("Started {} job", TASK_NAME);
 
-        getAvailableContainers().forEach(containerProcessor::process);
+        List<String> containers = getAvailableContainers();
+        Collections.shuffle(containers);
```

Comment:
I wish this returned a value... 😢

Reply:
We could try use something different this time - avoiding change in public method can just collect in private using `Collectors.collectingAndThen()`. Just a random idea